From 784201f67e611b722b8820b953548e81edba98c0 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Tue, 13 Mar 2012 15:09:29 +0800
Subject: ENGR00176629 [Thermal]Add 1.2G support

Fix bug of saving previous cpufreq not support
up to 1.2G freq. Max length set to 7 instead of 6.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 drivers/mxc/thermal/cooling.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mxc/thermal/cooling.c b/drivers/mxc/thermal/cooling.c
index 5a92b52..772f771 100644
--- a/drivers/mxc/thermal/cooling.c
+++ b/drivers/mxc/thermal/cooling.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2011 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright (C) 2011-2012 Freescale Semiconductor, Inc. All Rights Reserved.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -28,7 +28,7 @@
 
 #define	CPUx		"/sys/devices/system/cpu/cpu"
 #define MAX_GOVERNOR_NAME_LEN 12
-#define MAX_CPU_FREQ_LEN 6
+#define MAX_CPU_FREQ_LEN 7
 #define MAX_CPU_ONLINE_LEN 1
 
 /* Save the last hot point trigger temperature */
-- 
1.7.9.5

